diff -rupN TimeOrig//DateStrings.cpp Time/DateStrings.cpp
--- TimeOrig//DateStrings.cpp	2013-03-23 19:58:08 +0100
+++ Time/DateStrings.cpp	2013-06-21 16:50:07 +0200
@@ -38,7 +38,7 @@ const char monthStr10[] PROGMEM = "Octob
 const char monthStr11[] PROGMEM = "November";
 const char monthStr12[] PROGMEM = "December";
 
-PGM_P monthNames_P[] PROGMEM = 
+const char* const monthNames_P[] PROGMEM = 
 {
     "",monthStr1,monthStr2,monthStr3,monthStr4,monthStr5,monthStr6,
 	monthStr7,monthStr8,monthStr9,monthStr10,monthStr11,monthStr12
@@ -55,8 +55,8 @@ const char dayStr5[] PROGMEM = "Thursday
 const char dayStr6[] PROGMEM = "Friday";
 const char dayStr7[] PROGMEM = "Saturday";
 
-PGM_P dayNames_P[] PROGMEM = { dayStr0,dayStr1,dayStr2,dayStr3,dayStr4,dayStr5,dayStr6,dayStr7};
-char dayShortNames_P[] PROGMEM = "ErrSunMonTueWedThrFriSat";
+const char* const dayNames_P[] PROGMEM = { dayStr0,dayStr1,dayStr2,dayStr3,dayStr4,dayStr5,dayStr6,dayStr7};
+const char dayShortNames_P[] PROGMEM = "ErrSunMonTueWedThrFriSat";
 
 /* functions to return date strings */
 
